modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -27,28 +28,50 @@ local function new_{{classname}}(host, basePath, schemes)
     basePath = basePath or "{{{basePath}}}";
     schemes = schemes_map;
     default_scheme = default_scheme;
+    http_username = '';
```

Comment:
It will: but that's intentional. if the auth requires basic auth and they haven't provided a username/password, then an error *should* occur.